bookkeeper-server/bin/bookkeeper

```diff
@@ -71,13 +71,18 @@ else
 fi
 
 # exclude tests jar
-RELEASE_JAR=$(ls ${BK_HOME}/bookkeeper-server-*.jar 2> /dev/null | grep -v tests | tail -1)
+RELEASE_JAR=$(ls ${BK_HOME}/*bookkeeper-server-*.jar 2> /dev/null | grep -v tests | tail -1)
```

Comment:
If we are changing this script maybe we should write a note on upgrade instructions.
Maybe people usually replace only the contents of 'lib' directory in order to preserve configuration.

I am not an user of this script I am just asking

Reply:
I don't understand your comment here. This doesn't change any existing behaviour. This extends the lookup to adopt to bookkeeper-dist layout.

Reply:
Ok for existing users it will be enough to upgrade lib directory?

Reply:
Ok for existing users it will be enough to upgrade lib directory?

Reply:
what do you mean "upgrade"? you can't no simply copy lib directory, because dependencies have been changed. And why do you want to do that?

Reply:
sorry,
I never use the standard package, for upgrades we say "Upgrade the binary and configuration" in the guide.
what does this mean ?
for other products generally this requires to drop old jars and replace with new jars

anyway I am fine with the new script

Reply:
"upgrade the binary and configuration" means using the new package, you don't mixing files from different versions (I don't see any deployment methods would do that).

Reply:
yes sure I had a bad idea in mind.
usually you have the new package and you put the config files in order to use the data and the correct flags.